test/e2e/antreapolicy_test.go

```diff
@@ -1643,48 +1643,88 @@ func testANNPNestedGroupCreateAndUpdate(t *testing.T, data *TestData) {
 	executeTestsWithData(t, testCase, data)
 }
 
-// testBaselineNamespaceIsolation tests that an ACNP in the baseline Tier is able to enforce default namespace isolation,
+// testACNPBaselineIsolation tests that an ACNP in the baseline Tier is able to enforce default isolation,
 // which can be later overridden by developer K8s NetworkPolicies.
-func testBaselineNamespaceIsolation(t *testing.T) {
-	builder := &ClusterNetworkPolicySpecBuilder{}
-	nsExpOtherThanX := metav1.LabelSelectorRequirement{
-		Key:      "ns",
+func testACNPBaselineIsolation(t *testing.T) {
+	builder1 := &ClusterNetworkPolicySpecBuilder{}
+	podExpOtherThanC := metav1.LabelSelectorRequirement{
+		Key:      "pod",
 		Operator: metav1.LabelSelectorOpNotIn,
-		Values:   []string{getNS("x")},
+		Values:   []string{"c"},
 	}
-	builder = builder.SetName("acnp-baseline-isolate-ns-x").
+	builder1 = builder1.SetName("acnp-baseline-isolate-pod-c").
 		SetTier("baseline").
 		SetPriority(1.0).
 		SetAppliedToGroup([]ACNPAppliedToSpec{{NSSelector: map[string]string{"ns": getNS("x")}}})
-	builder.AddIngress(ProtocolTCP, &p80, nil, nil, nil, nil, nil, nil, nil, nil, nil,
-		nil, nil, []metav1.LabelSelectorRequirement{nsExpOtherThanX}, nil,
+	builder1.AddIngress(ProtocolTCP, &p80, nil, nil, nil, nil, nil, nil, nil, nil, nil,
+		nil, []metav1.LabelSelectorRequirement{podExpOtherThanC}, nil, nil,
+		nil, nil, crdv1beta1.RuleActionDrop, "", "", nil)
```

Comment:
A util function with so many arguments is difficult to use and error prone, especially when adjacent parameters are of the same type.
We should consider refactoring it in a more clear and flexible way some day.

Reply:
yes agree😶